fuzz/block_fuzz.go

```diff
@@ -191,7 +191,7 @@ func BeaconFuzzBlock(b []byte) {
 		return
 	}
 
-	if err := s.FuzzBeaconBlockSubscriber(ctx, msg); err != nil {
+	if err := s.FuzzBeaconBlockSubscriber(ctx, input.Block); err != nil {
```

Comment:
why is being changed, `msg` and `input.Block` arent the same.

Reply:
Would appreciate more guidance on this. Libp2p uses gogoproto which complicated some parts here

Reply:
So this issue came from our fuzz_exports.go changing from taking in a gogo.Message to a proto.Message, and libp2p is gogo messages, so not exactly sure how to handle this. Maybe we can keep using gogo for libp2p things and convert from there?

Reply:
Ok, looked at it again, undid these changes to keep results the same. Its fine if it uses gogo here, as this is code pertaining to libp2p, not our protobufs.

Reply:
Encountering problems though, would appreciate some assistance.

Reply:
@0xKiwi nice fix here. The FuzzBeaconBlockSubscriber was expecting a proto.Message and we were providing the incorrect message. Thanks!